policyengine_us/variables/gov/states/nm/tax/income/credits/cdcc/nm_cdcc_eligible.py

```diff
@@ -33,20 +33,12 @@ def formula(tax_unit, period, parameters):
         employment_eligible = where(joint, joint_eligible, head_has_earnings)
         # Filer can not receive tanf to be eligible
         receives_tanf = tax_unit.spm_unit("tanf", period) > 0
-        # Filers have to have modified gross of income below annual earning
-        # at double the federal minimum wage
+        # Filers have to have modified gross income at or below a limit
         nm_modified_gross_income = tax_unit("nm_modified_gross_income", period)
-        minimum_wage = parameters(period).gov.dol.minimum_wage
```

Comment:
We'd like to define the logic in accordance with the legal code, which is by minimum wage. Do we need this change?